### DISCUSSION TITLE
What is the monitor of lr_scheduler?

### DISCUSSION THREAD

**Discussion by Co1lin** [Lightning-AI/pytorch-lightning#8541] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi! I’m wondering what is `metric_to_track` for the parameter monitor  of a learning rate scheduler? I see the example that uses `"val_loss"` as its value, but I don’t know is `val_loss` a name of a variable (where should I declare this variable?) or a keyword that can be recognized by pytorch_lightning such that it will monitor the loss of the `validation_step`?

**Comment by talhaanwarch** [comment]
I think its the one you define in `self.log('val_loss ',loss, on_step=.., on_epoch=..,prog_bar=..)`

**Reply by Co1lin** [reply]
I think so. But I get a warning saying:

```shell
packages/pytorch_lightning/callbacks/model_checkpoint.py:610: LightningDeprecationWarning: Relying on `self.log('val_loss', ...)` to set the ModelCheckpoint monitor is deprecated in v1.2 and will be removed in v1.4. Please, create your own `mc = ModelCheckpoint(monitor='your_monitor')` and use it as `Trainer(callbacks=[mc])`.
```
I'm not sure whether this usage will be deprecated for scheduler's monitor too.

**Comment by talhaanwarch** [comment]
can you share your code snippet?

**Comment by ethanwharris** [comment]
This is a duplicate of #8554 so locking this discussion here.
